### PR TITLE
Make initial changes in docs

### PR DESCRIPTION
**Pull request by harshitbansal05**

### CODE REVIEW

**Comment by henrykironde** [comment]
Could you let me know what the issue is, I have done a quick test/ build and docs/recipes.rst seems to render well.

**Comment by harshitbansal05** [comment]
Idk why, but when the page for rdataretriever, recipes was not shown in the side list.

**Comment by henrykironde** [comment]
It would be nice if you can share some screenshots.

**Comment by harshitbansal05** [comment]
Screenshot 2019-06-27 at 1 10 56 AM

**Comment by harshitbansal05** [comment]
@henrykironde the side list does not contain recipes.

**Comment by henrykironde** [comment]
Clean your repo. I usually do a` git clean -fx ` (add -d  if you do not mind deleting IDE project files).
Make sure you are testing the correct branch.
`retriever reset all`
and `pip install . -U`
In my case `git clean -fxd` deletes the `docs/_build` directory
Try again with `make html`  and open in a new browser  or in incognito mode

File: docs/recipes.rst [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+.. _DataPackage: http://specs.frictionlessdata.io/data-packages/
+
+This functionality has now been moved to the ``retriever-recipes`` repository to separate the scripts from the core ``retriever`` functionalities to help with organization, maintenance, and testing. The `retriever recipes`_ repository thus holds all the scripts which were earlier shipped with ``retriever`` and also all the script adding/editing functionalities.
+
+.. _retriever recipes: https://github.com/weecology/retriever-recipes
+
+Installation
+------------
+
+The ``retriever-recipes`` project can simply be installed from Github via the following steps:
```
Comment (henrykironde) [inline docs/recipes.rst]: The ``retriever-recipes`` project can be installed from Github using the following steps

File: docs/recipes.rst [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+
+retriever-recipes
+-----------------
+
+The `Data Retriever`_ earlier used a simple CLI for developing new dataset scripts. This allowed users with no programming experience to quickly add most standard datasets to the Retriever by specifying the names and locations of the tables along with additional information about the configuration of the data. The script is saved as a JSON file, that follows the DataPackage_ standards.
+
+.. _Data Retriever: http://data-retriever.org
+.. _DataPackage: http://specs.frictionlessdata.io/data-packages/
+
+This functionality has now been moved to the ``retriever-recipes`` repository to separate the scripts from the core ``retriever`` functionalities to help with organization, maintenance, and testing. The `retriever recipes`_ repository thus holds all the scripts which were earlier shipped with ``retriever`` and also all the script adding/editing functionalities.
```
Comment (henrykironde) [inline docs/recipes.rst]: This functionality has been moved to the

**Comment by harshitbansal05** [comment]
@henrykironde done.

**Pull request merged**